tests/unit/test_bbox_conversion.py

```diff
@@ -0,0 +1,296 @@
+from google.cloud import documentai
+from google.cloud.documentai_v1.types import geometry
+from google.cloud.documentai_toolbox.converters.config import bbox_conversion, blocks
+
+
+def test_midpoint_in_bpoly():
+    box_a = geometry.BoundingPoly()
+    vertex_a = geometry.NormalizedVertex()
+    vertex_a.x = 2
+    vertex_a.y = 2
+    box_a.normalized_vertices = [vertex_a]
+    box_b = geometry.BoundingPoly()
+    vertex_b = geometry.NormalizedVertex()
+    vertex_b.x = 1
+    vertex_b.y = 1
+    vertex_b_max = geometry.NormalizedVertex()
+    vertex_b_max.x = 4
+    vertex_b_max.y = 4
```

Comment:
Could you load all of these values in the constructor instead of assigning later?